### PR TITLE
Edit2019

### PR DESCRIPTION
**Pull request by nancpond**

### CODE REVIEW

File: R/lazy-data.R [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#' @title a cluster sample data object
+#' @description A more complex data example of a cluster sample
+#' @docType data
+#' @format an object of class dataframe
+#' \itemize{
+#'   \item{clusterID}{cluster}
+#'   \item{bapa}{basal area per acre - attribute of interest}
+#'   \item{isUsed}{TRUE/FALSE}
+#' }
+#' @name clusterBAData
```
Comment (hrodmn) [inline R/lazy-data.R]: Maybe we could change the name to clusterBaData? I thought this was a BAD dataset at first 😛

File: tests/testthat/test_all_cluster.R [hunk 2]
```diff
@@ -1,15 +1,10 @@
...
-                       attr = c(1000, 1250, 950, 900, 1005, 1000, 1250, 950, 900, 1005, 1000,
-                                1250, 950, 900, 1005, 1000, 1250, 950, 900),
-                       isUsed = c(T, T, T, T, T, T, T, T, T, T, T, T, T, T, F, F, F, F, F))
-
 
 test_that("all cluster function handles simple cluster sample", {
   
-  expect_equal(summarize_all_cluster(dataPlot, element = TRUE, bernoulli = F), 
-               summarize_cluster(dataPlot, element = TRUE), tolerance = 0.1)
+  expect_equal(summarize_all_cluster(redData, attribute = 'volume', element = TRUE, bernoulli = F),
```
Comment (hrodmn) [inline tests/testthat/test_all_cluster.R]: I don't see this dataset (`redData`) documented in lazy-data.R, is it described somewhere else?

Comment (nancpond) [reply tests/testthat/test_all_cluster.R]: https://github.com/SilviaTerra/forestsamplr/blob/4b7a7aa0c86370acaca94a9f0b14af453ed13359/tests/testthat/test_twostage.R#L9

Comment (nancpond) [reply tests/testthat/test_all_cluster.R]: I'll find it in Avery and Burkhart and see what it is

Comment (nancpond) [reply tests/testthat/test_all_cluster.R]: Looks like it's just a simple cluster sample example dataset from a table in A&B. Easy enough to add to lazy-data

File: R/lazy-data.R (hunk 1 above)
Comment (nancpond) [reply R/lazy-data.R]: Changed

**Review by hrodmn — APPROVED** [review]

**Pull request merged**